src/org/skypro/skyshop/exeptions/BestResultNotFound.java

```diff
@@ -0,0 +1,8 @@
+package org.skypro.skyshop.exeptions;
+
+public class BestResultNotFound extends Exception {
```

Comment:
классы-исключения нужно называть с Exception на конце
```suggestion
public class BestResultNotFoundException extends Exception {
```

Reply:
Ок,рекомендации принял.Надо запомнить.НЕ знал.
Типа: "правила хорошего программиста и тона коммьюнити")))